doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/text/typography/TextTypographyPage.swift

```diff
@@ -0,0 +1,349 @@
+                        .font(.callout)
+                    
+                    Text("Footnote")
+                        .font(.footnote)
+                }
+                
+                Divider()
+                
+                // system font
+                VStack(spacing: 24) {
+                    Text("System font - 18, light, serif")
+                        .font(.system(size: 18, weight: .light, design: .serif))
+                    
+                    Text("System font - 18, medium, serif")
+                        .font(.system(size: 18, weight: .medium, design: .serif))
+                    
+                    Text("System font - 18, black, serif")
+                        .font(.system(size: 18, weight: .black, design: .serif))
+                    
+                    Text("System font - 18, light, rounded")
+                        .font(.system(size: 18, weight: .light, design: .rounded))
+                    
+                    Text("System font - 18, medium, rounded")
+                        .font(.system(size: 18, weight: .medium, design: .rounded))
+                    
+                    Text("System font - 18, black, rounded")
+                        .font(.system(size: 18, weight: .black, design: .rounded))
+                    
+                    Text("System font - 18, light, monospace")
+                        .font(.system(size: 18, weight: .light, design: .monospaced))
+                    
+                    Text("System font - 18, medium, monospace")
+                        .font(.system(size: 18, weight: .medium, design: .monospaced))
+                    
+                    Text("System font - 18, black, monospace")
+                        .font(.system(size: 18, weight: .black, design: .monospaced))
+                }
+                
+                Divider()
+                
+                // font weight
+                VStack(spacing: 24) {
+                    Text("fontWeight: regular")
+                        .fontWeight(.regular)
+                    
+                    Text("fontWeight: medium")
+                        .fontWeight(.medium)
+                    
+                    Text("fontWeight: semibold")
+                        .fontWeight(.semibold)
+                    
+                    Text("fontWeight: bold")
+                        .fontWeight(.bold)
+                    
+                    Text("fontWeight: heavy")
+                        .fontWeight(.heavy)
+                    
+                    Text("fontWeight: black")
+                        .fontWeight(.black)
+                    
+                    Text("fontWeight: light")
+                        .fontWeight(.light)
+                    
+                    Text("fontWeight: ultraLight")
+                        .fontWeight(.ultraLight)
+                }
+                
+                Divider()
+                
+                // font design
```

Comment:
As a matter of consistency, let's capitalize the first character of any comment.